template/index.ts

```diff
@@ -19,5 +19,7 @@ export function doSomeStuff(
   console.log(withThis);
   console.log(andThat);
   console.dir(andThose);
+  
```

Comment:
```suggestion

```
looks like you didn't run prettier. there are 2 extra spaces on this blank line which are not allowed. this suggested fix removes them, and allows the linter to pass.